libxaw will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxaw it will fail in the following way:

```
2024/11/12 23:12:34 INFO running step "test/pkgconf"
2024/11/12 23:12:34 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:34 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:34 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:34 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:34 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + basename /home/build/melange-out/libxaw-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + dev_pkg=libxaw-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + apk info -L libxaw-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + grep -q ^Name: usr/lib/pkgconfig/xaw7.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + basename usr/lib/pkgconfig/xaw7.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + lib_name=xaw7 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + echo usr/lib/pkgconfig/xaw7.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + grep -q '^usr/lib/pkgconfig/xaw7.pc$\|^usr/share/pkgconfig/xaw7.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + pkgconf --exists xaw7 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN + pkgconf --print-errors --exists xaw7 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN Package xproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN Perhaps you should add the directory containing `xproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN Package 'xproto', required by 'xaw7', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 WARN Package 'xt', required by 'xaw7', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:34 INFO pod 2ca22ae24e2eb3bddb4a378ae4495b2a05b6eeeefb91872603171962cb4adf3a terminated
2024/11/12 23:12:34 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:34 INFO   workspace dir: /home/user/tmp/melange-workspace-3925924521
2024/11/12 23:12:34 INFO   guest dir: /home/user/tmp/melange-guest-1143230008
2024/11/12 23:12:34 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxaw] Error 1
```

--- a/libxaw.yaml
+++ b/libxaw.yaml
@@ -53,7 +53,12 @@ subpackages:
     dependencies:
       runtime:
         - libxaw
+        - libxt-dev
+        - xorgproto
     description: libxaw dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxaw-doc
     pipeline:

--- a/libxaw.yaml
+++ b/libxaw.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxaw
   version: 1.0.16
-  epoch: 0
+  epoch: 1
   description: X Athena Widget Set
   copyright:
     - license: MIT
@@ -53,8 +53,6 @@ subpackages:
     dependencies:
       runtime:
         - libxaw
-        - libxt-dev
-        - xorgproto
     description: libxaw dev
     test:
       pipeline:

